spec/amazon_order/parsers/product_spec.rb

```diff
@@ -8,39 +8,31 @@ def ensure_fixture_filepath(path)
     path
   end
 
-  describe '#get_original_image_url' do
-    it "removes image options from url" do
-      parser = AmazonOrder::Parsers::Product.new(nil)
-      {
-        'https://images-fe.ssl-images-amazon.com/images/I/51lqodTD6KL.jpg' => 'https://images-fe.ssl-images-amazon.com/images/I/51lqodTD6KL.jpg',
-        'https://images-fe.ssl-images-amazon.com/images/I/51lqodTD6KL._UY250_.jpg' => 'https://images-fe.ssl-images-amazon.com/images/I/51lqodTD6KL.jpg',
-        'https://images-na.ssl-images-amazon.com/images/I/51i0hr6kccL._AC_US218_.jpg' => 'https://images-na.ssl-images-amazon.com/images/I/51i0hr6kccL.jpg',
-        'https://images-na.ssl-images-amazon.com/images/I/512S13U-XPL._SL500_SR103,135_.jpg' => 'https://images-na.ssl-images-amazon.com/images/I/512S13U-XPL.jpg',
-      }.each do |url, expected|
-        expect(parser.get_original_image_url(url)).to eql(expected)
-      end
-    end
-  end
-  
   Dir.glob("#{TARGET_DIR}/*html").each do |filepath|
     context "with file (#{filepath})" do
       parser = AmazonOrder::Parser.new(filepath)
-      before do
-        ensure_fixture_filepath(filepath)
-      end
+      before { ensure_fixture_filepath(filepath) }
 
       parser.orders.each do |order|
-        current_index = parser.orders.index(order)
-        context "for order #{current_index}" do
-          order.products.each do |product|
-            current_index = order.products.index(product)
-            context "for product #{current_index}" do
-              it 'has information' do
-                expect(product.title).to be_present
-                expect(product.path).to match(%r{\A/gp/product/})
-                expect(product.content).to be_present
-                expect(product.image_url).to match(%r{/images/I/[^.]+\.jpg})
-                expect(product.fetched_at).to be_present
+        context "for order #{order.order_number}" do
```

Comment:
This spec harness is getting a little bit crazy.... 

Perhaps the fixtures should be broken into separate files at each level so we can test them more cleanly?

I'm not sure what the right approach is here. This works for now, but it isn't pretty.

Reply:
Shipment isn't in common with all pages.
Testing only with 'spec/fixtures/files/order-2018-p1-20180129141532-contains-home-service.html' is enough for me.

Reply:
It's OK to replace `Dir.glob("#{TARGET_DIR}/*html").each do |filepath|` with `filepath = 'spec/fixtures/files/order-2018-p1-20180129141532-contains-home-service.html'`.